2.6.md

```diff
@@ -2963,6 +2988,40 @@ Information on how often and what triggers an ad slot being refreshed.
   </tr>
 </table>
 
+### 3.2.35 - Object: DurFloors <a name="objectdurfloors"></a>
+
+Allows publishers to specify prices for audio or video creatives of different durations. There are no explicit constraints on the defined ranges, nor guarantees that they don't overlap. This is between the seller and buyer to sort out. In simple terms, this object is used by sellers to provide pricing information to buyers by duration. For example: 1-15 seconds at a floor of $5; 16-30 seconds at a floor of $10, > 31 seconds at a floor of $20.
```

Comment:
Change this: This is between the seller and buyer to sort out. to *something like* : These negotiations are to be negotiated directly between the seller and the buyer _a priori_

Reply:
@hillslatt updated language based on your feedback. Let me know if this looks good now.